dali/pipeline/data/tensor_list.h

```diff
@@ -44,8 +44,11 @@ class Tensor;
 template <typename Backend>
 class DLL_PUBLIC TensorList : public Buffer<Backend> {
  public:
-  DLL_PUBLIC TensorList()
-    : layout_(DALI_NHWC) {}
+  DLL_PUBLIC TensorList() {}
```

Comment:
Do you need to repeat DLL_PUBLIC on functions when class already has it? I don't think so...